hubble/Makefile

```diff
@@ -1,31 +1,43 @@
-GO := go
-GO_BUILD_FLAGS =
-GO_TEST_FLAGS =
-GO_BUILD = CGO_ENABLED=0 $(GO) build $(GO_BUILD_FLAGS)
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Authors of Hubble
+
+.DEFAULT_GOAL := all
```

Comment:
nit: IMO this is adding more confusion that helping, I would guess that most makefile developers know that the default target is the first one, but don't know the GNU Make specific special variables. Not feeling too strongly about it though.

Reply:
Good point, removed it, I'd brought this out of consistency with other makefiles like `daemon/Makefile` or `operator/Makefile` but agreed it's not necessary